### PR TITLE
All: Translation: Update nl_NL.po

### PR DESCRIPTION
**Pull request by moun3m**
fixed some mistranslations and added some missing

### CODE REVIEW

File: packages/tools/locales/nl_NL.po [hunk 1]
```diff
@@ -3302,9 +3301,8 @@ msgid "Downloaded and encrypted"
 msgstr "Gedownload en versleuteld"
 
 #: packages/lib/services/ReportService.js:180
-#, fuzzy
 msgid "Created locally"
-msgstr "Lokale items gecreëerd: %d."
+msgstr "Lokale items gecreëerd: %d"
```
Comment (tessus) [inline packages/tools/locales/nl_NL.po:3305]: The source does not have a `%d`. Why does the translation? Something is wrong here.

File: packages/tools/locales/nl_NL.po [hunk 2]
```diff
@@ -3966,7 +3963,7 @@ msgstr "N"
 
 #: packages/app-cli/app/app.js:137
 msgid "y"
-msgstr "j"
+msgstr "J"
```
Comment (tessus) [inline packages/tools/locales/nl_NL.po:3966]: The source string has lowercase `y`. Why do you use an uppercase character for the translation?

**Review by tessus — CHANGES_REQUESTED** [review]

**Comment by tessus** [comment]
Also, please do not commit the `.mo` file.

**Comment by laurent22** [comment]
Closing for now. Please let us know if you'd like to update the pull request.

File: packages/tools/locales/nl_NL.po (hunk 1 above)
Comment (moun3m) [reply packages/tools/locales/nl_NL.po:3305]: The source does have "Created local items: %d."

Comment (tessus) [reply packages/tools/locales/nl_NL.po:3305]: The source string for this translation is `Created Locally` (line 3304).

The translation you are talking about is on line 2994.

Comment (tessus) [reply packages/tools/locales/nl_NL.po:3305]: In Poedit search for the source string `Created locally` and you'll see what I mean.

Comment (tessus) [reply packages/tools/locales/nl_NL.po:3305]: No worries. Just make the changes in the other PR, add the commit and I can merge...

**Pull request closed without merging**